docs/docs/how-tos/How to install and configure Keycloak.md

```diff
@@ -0,0 +1,151 @@
+---
+id: configuring-keycloak
+title: How to configure Keycloak and add new users
+description: |
+  This is a how-to for configuring Keycloak and add new users.
+---
+
+## Configuring Keycloak
+
+[Keycloak](https://www.keycloak.org/) is the name of the open-source user management software that's automatically deployed within Nebari. It's used to store the database of all
+users in your Nebari instance, and can provide connectivity to other services such as GitHub/Auth0 single sign-on.
+
+As with many user admin tools, in Keycloak you start with an initial login (`root`) that has the ability to administer and create new users. The `root` user is a Keycloak-specific user. It can only be used to login and manage the Keycloak identity management section of Nebari. It's not a user of the wider Nebari data science platform.
```

Comment:
we already mentioned this shortly in the deploy docs (almost certain about this...), so If possible, I would like this to be the source of truth for Keycloak.

Reply:
I would say both - since I assume folks will not linearly follow the docs, also redundancy. Someone updating keycloak configs might not be the same person deploying nebari